src/thunderbots/software/ai/world/game_state.h

```diff
@@ -0,0 +1,297 @@
+#pragma once
+
+#include "geom/point.h"
+#include "util/refbox_constants.h"
+
+/**
+ * @brief Holds the state of the game according to the referee
+ *
+ * @details Contains information on what period of the game it is, what type of
+ * play to run, team scores, time remaining, etc.  During normal gameplay, the
+ * information in this class is received from the
+ * [ssl-refbox](https://github.com/RoboCup-SSL/ssl-refbox)
+ * program over the network.
+ *
+ * Taken from https://github.com/RoboJackets/robocup-software/
+ */
+class GameState
+{
+   public:
+    enum State
```

Comment:
Would be good to include a comment explaining that the `State` and `RestartReason` enums are subsets of the refbox `Command` enum that combine in different ways to represent all the `RefBoxGameStates`

AKA the permutations of combining the `State` and `RestartReason` enums gives the `RefBoxGameStates`